lms/templates/courseware/course_about.html

```diff
@@ -186,8 +186,19 @@
           </a>
           % if sneakpeek_allowed and not regularly_registered:
             <a href="#" class="course_sneakpeek">
-              <span class="sneakpeek" title="${_("By exploring the course, you are agreeing to our Terms of Service. Please read them carefully.")}">${_("Explore Course")} *</span>
+              <span class="sneakpeek">${_("Explore Course")} *</span>
             </a>
+            <section class="unauth-tos" id="unauth-tos">
```

Comment:
Assuming this is the same footer we're using elsewhere, we should move this into an separate template/fragment and include it where we need it; that ensures it's always displayed consistently.